### PR TITLE
feat(frontend): improve homepage

### LINKED ISSUE #439: Improve Homepage Styles
https://guide.eoscostarica.io/

image

- [ ] Dark mode 
- [ ] blog posts
- [ ] sections links 
- [ ] fonts 
- [ ] mobible view

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- resolves #439 and #168 

### Steps to test
1. Go to the Homepage
2. See new changes

#### CheckList
- [x] Images passing by carousel with new images
- [x] Change styles to cards
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

File: src/pages/BlogCard/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { CustomArticle } from "./CustomArticle";
+
+import "../../css/homepage.css";
+
+const articlesList = [
+  {
+    title: "Get to Know EOS Costa Rica",
+    body: "Learn more about our history, team and projects we carry out.",
+    href: "https://medium.com/@eoscostarica/get-to-know-eos-costa-rica-f91f5b0bb7c4",
+    //img: "https://miro.medium.com/max/700/1*x7OW2wAhH1YapNVZl_-SEA.png"
```
Comment (AngeloCG97) [inline src/pages/BlogCard/index.js:13]: Remove this comment please

File: src/pages/BlogCard/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    img: "/img/cards-icons/blog1.png"
+  },
+  {
+    title: "You Should Participate in an Open-Source Project",
+    body: "We show some important aspects for which it should be dealt with in open source projects.",
+    href: "https://medium.com/@eoscostarica/you-should-participate-in-an-open-source-project-c259f8759d41",
+    //img: "https://miro.medium.com/max/875/1*_eTme4t7HJfE7aeeyd3kxQ.jpeg"
```
Comment (AngeloCG97) [inline src/pages/BlogCard/index.js:20]: Remove this comment please

File: src/pages/BlogCard/index.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    img: "/img/cards-icons/blog2.png"
+  },
+  {
+    title: "What Is Blockchain?",
+    body: "We explain, with examples, what a blockchain is and how it works.",
+    href: "https://medium.com/@eoscostarica/what-is-blockchain-an-introduction-9535ed3e6005",
+    //img: "https://miro.medium.com/max/875/1*ohmD1qq4dAUOu4G-gXtaqA.jpeg"
```
Comment (AngeloCG97) [inline src/pages/BlogCard/index.js:27]: Remove this comment please

File: src/pages/BlogCard/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      >
+        {
+          articlesList.map(({ title, body, href, img }) =>
+            <Grid key={title} item xs={12} sm={6} md={6} lg={4} container display="flex" justifyContent="center">
+              <CustomArticle
+                title={title}
+                body={body}
+                href={href}
+                img={img}
+              ></CustomArticle>
```
Comment (AngeloCG97) [inline src/pages/BlogCard/index.js:59]: Please change  

```
<CustomArticle
                title={title}
                body={body}
                href={href}
                img={img}
              ></CustomArticle>
```
to

```
            <CustomArticle
                title={title}
                body={body}
                href={href}
                img={img}
              />
```

File: src/pages/heroSection/ImagesCarousel.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        return () => carousel.stop();
+    }, []);
+    
+    return (
+        <Box key={background} className={`slideClass ${fade}`} display={index > 0 ? "none" : "block"} >
+            <Box className={numbertext}>
+                {index + 1} / 3
+            </Box>
+            <img src={useBaseUrl(background)} width="100%" style={{
+                minHeight: "175px",
```
Comment (AngeloCG97) [inline src/pages/heroSection/ImagesCarousel.js:51]: Please remove inline style code

File: src/pages/heroSection/index.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+      </Box>
+      <Box display="flex" justifyContent="center" width="100%" position="absolute" bottom={mdDown ? "55px" : "6vw"}>
+        <Button className={button} href="/docs/engineering-culture" >
+          <Typography className={textButton}>
+            GET STARTED
+          </Typography>
+        </Button>
+      </Box>
+      <br />
+      <Box display="flex" justifyContent="center" position="absolute" bottom="40px" width="100%">
```
Comment (AngeloCG97) [inline src/pages/heroSection/index.js:61]: In bottom prop you can use material ui dimensions, for example bottom={5}

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]
Review this comments please

File: src/pages/BlogCard/index.js (hunk 1 above)
Comment (JAMares) [reply src/pages/BlogCard/index.js:13]: Fixed up

File: src/pages/BlogCard/index.js (hunk 3 above)
Comment (JAMares) [reply src/pages/BlogCard/index.js:27]: Fixed up

File: src/pages/BlogCard/index.js (hunk 4 above)
Comment (JAMares) [reply src/pages/BlogCard/index.js:59]: Fixed up

File: src/pages/heroSection/index.js (hunk 6 above)
Comment (JAMares) [reply src/pages/heroSection/index.js:61]: Fixed up

File: src/pages/BlogCard/index.js (hunk 2 above)
Comment (JAMares) [reply src/pages/BlogCard/index.js]: Fixed up

File: src/pages/heroSection/ImagesCarousel.js (hunk 5 above)
Comment (JAMares) [reply src/pages/heroSection/ImagesCarousel.js]: Fixed up

**Review by Torresmorah — COMMENTED** [review]
maybe, jpg or png images could be changed to webp format to reduce their file size

**Comment by JAMares** [comment]
Requested changes fixed

File: src/css/homepage.css [hunk 7]
```diff
@@ -2,184 +2,205 @@
...
-  margin-bottom: 50px;
+/* This change customCard header */
+.cardroot:hover .header-transition {
+  background-color: #2d9cdb;
 }
 
-.section-container{
-  margin-top: 50px;
-  margin-bottom: 50px;
+/* This change customCard header */
```
Comment (xavier506) [inline src/css/homepage.css:70]: This change**s** customCard header

File: src/pages/heroSection/index.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+      </Box>
+      <Box
+        display="flex"
+        justifyContent="center"
+        width="100%"
+        position="absolute"
+        bottom={mdDown ? "95px" : "12vw"}
+      >
+        <Typography className={textImage}>
+          Welcome to our open-source developers portal. Begin your journey to
```
Comment (xavier506) [inline src/pages/heroSection/index.js:68]: lets include **web3** as an additional keyword here , like this:

Welcome to our web3 developers portal...

File: src/pages/heroSection/index.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Box, Button, Typography } from "@material-ui/core";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+import { heroStyles, heroMdStyles, heroSmStyles } from "./styles";
+import { ImagesCarousel } from "./ImagesCarousel";
+
+const imagesList = [
+  {
+    background: "/img/cards-icons/EOS-Previews-2.webp",
```
Comment (xavier506) [inline src/pages/heroSection/index.js:10]: can we change the order of hero images to leave the one with NFT text for last?

I like the NFT image but I feel the NFT text interferes with the header text a bit

image

@Edwinjrc  any thoughts?

**Review by xavier506 — CHANGES_REQUESTED** [review]
Looks Great! Amazing Job! 🎖️ 

Please see my comments on minor changes

File: src/css/homepage.css (hunk 7 above)
Comment (xavier506) [reply src/css/homepage.css:70]: minor gramatical error in comment

File: src/pages/heroSection/index.js (hunk 9 above)
Comment (warnerHurtado) [reply src/pages/heroSection/index.js]: Image order changed

File: src/pages/heroSection/index.js (hunk 8 above)
Comment (warnerHurtado) [reply src/pages/heroSection/index.js]: Ready

File: src/css/homepage.css (hunk 7 above)
Comment (warnerHurtado) [reply src/css/homepage.css]: Ready

**Review by xavier506 — DISMISSED** [review]
LGTM

File: src/pages/CustomArticle/CustomArticle.js [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+        responsiveBody,
+        responsiveBlueBoxTitle,
+        normalBlueBoxTitle, } = dimentionsClass();
+
+    return (
+        <Box border={1} width={smDown ? "256px" : "360px"} height={smDown ? "300px" : "412px"} className="cardroot"
+            marginBottom={smDown ? "8px" : ""}
+        >
+            <Box>
+                <img src={useBaseUrl(img)} loading="lazy" width="100%" style={{ borderBottom: "1px solid black" }} />
```
Comment (AngeloCG97) [inline src/pages/CustomArticle/CustomArticle.js:30]: Please remove inline style code

File: src/pages/heroSection/ImagesCarousel.js [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+    const carousel = new Carousel();
+    carousel.start();
+    return () => carousel.stop();
+  }, []);
+
+  return (
+    <Box
+      key={background}
+      className={`slideClass ${fade}`}
+      display={index > 0 ? "none" : "block"}
```
Comment (AngeloCG97) [inline src/pages/heroSection/ImagesCarousel.js:50]: You can use condition and not display for example


```
{(index > 0) &&
<Box
      key={background}
      className={`slideClass ${fade}`}
     >
      <Box className={numbertext}>{index + 1} / 3</Box>
      <img id="imgCarousel-id" src={useBaseUrl(background)} width="100%" />
    </Box>
}
```

File: src/pages/CustomArticle/CustomArticle.js (hunk 10 above)
Comment (JAMares) [reply src/pages/CustomArticle/CustomArticle.js]: fixed up

File: src/pages/heroSection/ImagesCarousel.js (hunk 11 above)
Comment (JAMares) [reply src/pages/heroSection/ImagesCarousel.js]: fixed up

File: src/pages/heroSection/index.js (hunk 9 above)
Comment (Edwinjrc) [reply src/pages/heroSection/index.js]: Yeah I think that image has too much information. We just wanted to try it, but I think I will talk about it with Johanna and see if we can change it for a more appropiate one.

**Review by xavier506 — CHANGES_REQUESTED** [review]
please see codefactor comments

**Comment by JAMares** [comment]
Requested changes codefactor fixed

**Review by xavier506 — DISMISSED** [review]
LGTM

**Review by JAMares — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**